requirements.txt

```diff
@@ -17,6 +17,7 @@ appdirs
 palettable
 dash
 dash-bootstrap-components
+quilt3
 tqdm
-hdbscan
```

Comment:
hdbscan will only be necessary when the user needs the fuctions that rely on it?

Reply:
the last commit should handle that